containers/images/hawkular-mock-agent/resources.py

```diff
@@ -18,11 +19,29 @@
 """
 
 import json
+import os
 import requests
 
 class Resources (object):
 
     @staticmethod
+    def get_headers():
+        headers = {'Content-type': 'application/json', 'Accept': 'application/json'}
+        return headers
+
+    @staticmethod
+    def get_milliseconds_request():
+        return int(os.environ['MILLISECONDS_REQUEST'])
```

Comment:
for the os.environ variables, could you maintain the default values if those are not defined ?

Reply:
Added on fb96e0c and 853f7ec

Reply:
@lucasponce Can you review the new commits?